authority/tls.go

```diff
@@ -204,7 +193,7 @@ func (a *Authority) Renew(oldCert *x509.Certificate) ([]*x509.Certificate, error
 		return nil, errs.Wrap(http.StatusInternalServerError, err,
 			"authority.Renew; error parsing new server certificate", opts...)
 	}
-	caCert, err := x509.ParseCertificate(issIdentity.Crt.Raw)
+	caCert, err := x509.ParseCertificate(a.x509Issuer.Raw)
```

Comment:
Why not just return `a.x509Issuer` as caCert as you're doing above in the `Sign` method.

Reply:
Fixed with 05cc143